tests/testthat/test-merge-clin.R

```diff
@@ -0,0 +1,32 @@
+
+# Setup ----
+clin_file <- system.file("cli/merge", "meta.csv", package = "SomaDataIO",
+                          mustWork = TRUE)
+clin_df <- read.csv(clin_file, header = TRUE, colClasses = c(SampleId = "character"))
+apts <- withr::with_seed(123, sample(getAnalytes(example_data), 2L))
+adat <- head(example_data, 9L) |> dplyr::select(SampleId, all_of(apts))
+
+test_that("merge_clin() errors on bad `clin_data` argument", {
```

Comment:
This test description should be modified to something like: `"merge_clin() generates expected merged output"`

Reply:
Ohhhhhh, good catch. My copy-paste game was a little too rigorous!